src/joy.cpp

```diff
@@ -20,6 +20,11 @@
 
 
 #include "fcu_common/joy.h"
+#include "geometry_msgs/Pose.h"
+#include "geometry_msgs/Twist.h"
```

Comment:
I think that these are included already by gazebo_msgs/ModelState.h  If they are, there is no reason to include them again.